AutoGrade/models.py

```diff
@@ -171,12 +174,13 @@ class Submission(models.Model):
     passed          = models.IntegerField(default=0)
     failed          = models.IntegerField(default=0)
     publish_date    = models.DateTimeField('date published', default=datetime.now)
+    late_sub       = models.BooleanField(default=False)
```

Comment:
Don't think we need this. If it is a late submission, we can find out on runtime by comparing the submission date with the assignment due_date.

Reply:
I didn't read first comment on #9 , I just read last two comments and start working with what I understand, that's why it happened.